client/express-checkout/blocks/index.js

```diff
@@ -1,21 +1,40 @@
 /**
  * Internal dependencies
  */
-import { PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT } from '../../checkout/constants';
+import { PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT } from 'wcpay/checkout/constants';
+import { getConfig } from 'wcpay/utils/checkout';
+import ApplePayPreview from './components/apple-pay-preview';
 import ExpressCheckoutContainer from './components/express-checkout-container';
-import { getConfig } from '../../utils/checkout';
-import ApplePayPreview from './apple-pay-preview';
 
-const expressCheckoutElementPaymentMethod = ( api ) => ( {
-	name: PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT,
+const expressCheckoutElementApplePay = ( api ) => ( {
+	paymentMethodId: PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT,
+	name: PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT + '_applePay',
+	content: (
+		<ExpressCheckoutContainer api={ api } expressPaymentMethod="applePay" />
+	),
+	edit: <ApplePayPreview />,
+	supports: {
+		features: getConfig( 'features' ),
+	},
+	canMakePayment: () => {
+		if ( typeof wcpayExpressCheckoutParams === 'undefined' ) {
+			return false;
+		}
+
+		return true;
+	},
+} );
+
+const expressCheckoutElementGooglePay = ( api ) => ( {
+	paymentMethodId: PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT,
+	name: PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT + '_googlePay',
 	content: (
 		<ExpressCheckoutContainer
 			api={ api }
-			stripe={ api.loadStripe( true ) }
+			expressPaymentMethod="googlePay"
 		/>
 	),
 	edit: <ApplePayPreview />,
```

Comment:
Should we create a `GooglePayPreview` as well as part of this PR, so it doesn't look like this in the blocks editor?

<img width="1402" alt="Duplicate edit block preview" src="https://github.com/Automattic/woocommerce-payments/assets/5509901/2bbb4d57-cde0-4732-b035-7b2289bfcc0e">

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/9047/commits/500e17ba717f887a08eb9ad623e0a544c5dd72e8